x-pack/plugins/apm/server/saved_objects/apm_indices.ts

```diff
@@ -36,6 +37,25 @@ export const apmIndices: SavedObjectsType = {
         defaultMessage: 'APM Settings - Index',
       }),
   },
+  modelVersions: {
+    '1': {
+      changes: [],
+      schemas: {
```

Comment:
What happens if we don't register it? Is this something we should do even if we don't plan any changes here?

Reply:
If you don't plan any changes it's not necessary.

The particularity is that this schema is set on the *prior* version. E.g if at some point you realise that you would need to add new fields, you would be forced to add the `forwardCompatibility` on your current model version, and make sure that this change is pushed to production before effectively introduce your new version (which is why we encourage to do it sooner rather than later)/